src/openapi_parser/builders/path.py

```diff
@@ -42,4 +42,13 @@ def _build_path(self, url: str, data: dict) -> Path:
             if method.value in data
         ]
 
+        if attrs.get("parameters"):
```

Comment:
> @manchenkoff added test cases for extensions for the path parameters inheritance the cases are already there in path builder test and runner test. LMK if you want me to add some other cases

I've checked [path builder test](https://github.com/manchenkoff/openapi3-parser/blob/c0c91ec77b94cb0820f068789f6bd715e8ae07fd/tests/builders/test_path_builder.py#L36), but it looks like Operation  doesn't have parameters from Path, is it expected behavior for this your change?

Reply:
I noticed this as well, and it appears that the same operation object is present in both the expected and actual results. I debugged it and saw same parameter in operation as well (3 case). Let me try to understand it better.

Reply:
@manchenkoff 
In fact, the test is somewhat challenging :-) The same operation_object (reference) is utilized for the mock_builder and as the expected object. Therefore, when the path builder constructs the actual object, it updates the parameters for both. 
I can fix the test. (the behavior is expected)

Reply:
yikes 😄 yes, please update the test once you have time

Reply:
done